integration/go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/go-test/deep v1.0.7 // indirect
 	github.com/jedib0t/go-pretty v4.3.0+incompatible
 	github.com/logrusorgru/aurora v2.0.3+incompatible // indirect
-	github.com/onflow/cadence v0.19.1-0.20210920215340-75c54c6c01eb
+	github.com/onflow/cadence v0.20.0-beta10
```

Comment:
We can now use v0.20.0 here, which is the same as v0.20.0-beta10. We can also do that once this PR is merged

Reply:
yeah I was thinking to do it in another PR